docs/cudf/source/developer_guide/cudf_pandas.md

```diff
@@ -3,8 +3,16 @@ The use of the cuDF pandas accelerator mode (`cudf.pandas`) is explained [in the
 The purpose of this document is to explain how the fast-slow proxy mechanism works and document internal environment variables that can be used to debug `cudf.pandas` itself.
 
 ## fast-slow proxy mechanism
-`cudf.pandas` works by wrapping each Pandas type and its corresponding cuDF type in a new proxy type also known as a fast-slow proxy type.
-The purpose of proxy types is to attempt computations on the fast (cuDF) object first, and then fall back to running on the slow (Pandas) object if the fast version fails.
+The core of `cudf.pandas` is implemented through proxy types defined in [`fast_slow_proxy.py`](https://github.com/rapidsai/cudf/blob/57aeeb78d85e169ac18b82f51d2b1cbd01b0608d/python/cudf/cudf/pandas/fast_slow_proxy.py#L869), which link a pair of "fast" and "slow" libraries.
```

Comment:
Did you intend to link to `_fast_slow_function_call` specifically? If so, I would clarify `which link a pair of "fast" and "slow" libraries` as `which executes a "fast" library and a "slow" library if the "fast" library fails`.

Reply:
No that that should be a link to the file, thanks